### ISSUE TITLE
Hand out proposals for review to encourage uniform review counts

### ISSUE THREAD

**Issue by mw44118** [pyohio/pyohio#44]
Right now, reviewers see the list of proposals sorted by submission date.  And most reviewers start at the top and never review all the proposals.

So the talks / tutorials submitted first show up first, and get a ton of votes.  Some talks at the bottom of the list get just one or two votes.

We need to add some text to the reviewer page asking reviewers to seek out the lesser-reviewed talks.  Also we should sort the talks so that the ones with the least votes show up first.

**Comment by tylerdave** [comment]
Agreed. I think there's a lot of room for improvement with the review process but a simple improvement for now would be sorting by lowest review count, then oldest submission by default.
